usdr-gost.code-workspace

```diff
@@ -26,6 +26,6 @@
 		// competing user settings and workplace settings. We use VSCode's "codeActionsOnSave"
 		// to manage linting/fixing, but users can have other settings via "formatOnSave".
 		// If both are on, the editor can "fix" things that are immediately flagged as lint failures.
-		"editor.formatOnSave": false,
+		"editor.formatOnSave": true,
```

Comment:
I don't think we want to check this change in. Were you not able to get automatic linting on save working without this change? 

I'd love to look into that a bit more with you... the `codeActionsOnSave` is supposed to supersede the `formatOnSave` setup in VSCode I believe, but the setup is finicky, and I didn't have many users to test out the settings with when I was changing them last...

Reply:
whoops sorry this was careless, I was playing with settings and accidentally committed this. my bad.